DRAFT.md

```diff
@@ -892,9 +899,9 @@ constexpr indirect& operator=(indirect&& other) noexcept(
   `std::allocator_traits<Alloc>::propagate_on_container_move_assignment` is
   `true` and `alloc != other.alloc` then the allocator needs updating.
 
-  If `other` is valueless, `*this` becomes valueless and the owned value in this,
-  if any, is destroyed using `allocator_traits<allocator_type>::destroy` and then
-  deallocated using `allocator_traits<allocator_type>::deallocate`.
+  If `other` is valueless, `*this` becomes valueless and the owned value in
+  this, if any, is destroyed using `allocator_traits<allocator_type>::destroy`
+  and then deallocated using `allocator_traits<allocator_type>::deallocate`.
```

Comment:
```suggestion
  and then the storage is deallocated using `allocator_traits<allocator_type>::deallocate`.
```

Since you already said the allocator is used, maybe all these cases could just say "the storage is deallocated" and not mention how. There's no other way to do it except the allocator's `deallocate` member.

Reply:
Oh, especially as it should really be `allocator_traits<allocator_type>::rebind_traits<T>::deallocate`  for `polymorphic`, which is not correct below!

So not mentioning `deallocate` by name avoids that problem.